app/javascript/packs/banner_editor.jsx

```diff
@@ -975,13 +975,13 @@ export default class BannerEditor extends React.Component {
                   </DropdownToggle>
                   <TextInput
                     link
-                    onChange={e => this.updateCurrentLink(e)}
-                    value={this.state.currentLink}
+                    onChange={e => this.updateCurrentUsername(e)}
```

Comment:
Better if we just do this, as the current line creates an anonymous function which will be re-created every render
```suggestion
                    onChange={this.updateCurrentUsername}
```